### PR TITLE
add more facebook groups and a volunteering opportunity

### PR DESCRIPTION
**Pull request by polina-munoz**
# hacktoberfest 👻 🎃 🍺

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,14 +41,18 @@ List of organizations for women in tech accessible from the bay area.
...
+* [Ladies Storm Hackathons General](https://www.facebook.com/groups/LadiesStormHackathons/)/[Bay Area](https://www.facebook.com/groups/LadiesStormHackathonsBay/)/[Jobs and Internships](https://www.facebook.com/groups/LSHJobs/)
  * Based off of the popular Facebook group [Hackathon Hackers](https://www.facebook.com/groups/hackathonhackers/), this group's MO is to help close the gender gap in hackathons. There's also a Bay Area variant (linked above)
 * [Russian-Speaking Women in Tech Silicon Valley](https://www.facebook.com/groups/RussianWomenInTech)
 * [EdTechWomen](https://www.facebook.com/groups/885534781541043/) - [SF](https://www.facebook.com/groups/1062156953802924/)
 * Women of Color in Tech - [Bay Area](https://www.facebook.com/groups/WOCinbay/?ref=br_rs)
 * [Women in Gaming](https://www.facebook.com/groups/396715327030958/?ref=br_rs)
 * [San Francisco Women in Tech](https://www.facebook.com/groups/201493976856135)
 * WOMEN GET IT DONE - [SF](https://www.facebook.com/groups/womengetitdone/)
+* [Silicon Valley Women](https://www.facebook.com/groups/SVWomen/)
```
Comment (maggiewhite) [inline README.md:52]: Would you mind adding a line or two describing the purpose of the group?

File: README.md [hunk 2]
```diff
@@ -41,14 +41,18 @@ List of organizations for women in tech accessible from the bay area.
...
+* [Female Founders Conference](https://www.facebook.com/groups/femalefoundersconference/)
+ * Y Combinator's community of female founders, this group is great for supporting and meeting other entrepreneurs and leaders in tech
+* [Girls in Tech - San Francisco](https://www.facebook.com/groups/55771429119/)
```
Comment (maggiewhite) [inline README.md:56]: Same, please add a short description of the group

**Review by maggiewhite — CHANGES_REQUESTED** [review]

**Comment by polina-munoz** [comment]
@acqcolor sounds good! I just added descriptions for all of the groups, lmk if those work.

**Review by maggiewhite — APPROVED** [review]
Looks great! Thanks for the addition.

**Pull request merged**